libgalois/include/katana/GraphTopology.h

```diff
@@ -605,6 +607,156 @@ class KATANA_EXPORT BasicTopologyWrapper : public GraphTopologyTypes {
   const Topo* topo_ptr_;
 };
 
+/// filter nodes and edges based on their type values
+/// and creates a new local graph based on the filtered nodes and edges
```

Comment:
Drop the word "local". All graphs are local. It can be confusing. Instead you can use "projected"